uv.lock

```diff
@@ -1,13 +1,13 @@
 version = 1
 requires-python = ">=3.10"
 resolution-markers = [
-    "(python_full_version < '3.11' and platform_machine != 'aarch64' and sys_platform == 'linux') or (python_full_version < '3.11' and sys_platform != 'darwin' and sys_platform != 'linux')",
+    "(python_full_version < '3.11' and platform_machine != 'aarch64' and sys_platform != 'darwin') or (python_full_version < '3.11' and sys_platform != 'darwin' and sys_platform != 'linux')",
```

Comment:
are these changes to uv.lock ok?

Reply:
Hum I think the uv.lock might be out of date.

Reply:
Ok fixed, this `uv.lock` is strange, I feel like I'm a different result each time I run it...

Good catch!